### ISSUE TITLE
absolute bafoons

### ISSUE THREAD

**Issue by Duckmc34** [darktrovx/interact#28]
idk why you decided to be this stupid or dumb, but somehow you did, whom ever decided to add the feature to "combine" interact things, you're an idiot, you didn't even add a config for it, let's just remove the ability to have different interactives if they are close to eachother.... bafoons..

**Comment by Duckmc34** [comment]
Completely agree, it would be if people like these could thing for themselves, but i guess you can't expect that much.

**Comment by Duckmc34** [comment]
you're completely right, they should atleast have added some sort of a config file to disable this shit, they spent all this time on this project, but makes it shit with one feature!! HAHAHA!!!!!! !!!!!! ! ! ! `!INF! HA !

**Comment by Duckmc34** [comment]
HAHA!!!! CURRENTLY HAVING THE SAME ISSUE""!"!! WHEN I GO CLOSE TO 2 DOORS, ALL THE OPTIONS FROM BOTHS DOORS APPEAR AT ONE DOOR ONLY??!! (WHAT YOU ASK"?"") OOH THAT'S JUST A STUPID BAFOONISHLY FEATURE!!!


OH DOES THIS ONLY DO THIS WEIRD THING WITH DOORS!?!? HMMMMM NO!


![OKOKOKOKOK](https://github.com/darktrovx/interact/assets/26554440/63c0a11c-4535-4eef-bbd8-95b27f45133c)







function nusser(doors)
	for i = #doorsliste, 1, -1 do
		local doorsvin = doorsliste[i]
		table.remove(doorsliste, i)
		exports.interact:RemoveInteraction(doorsvin)
	end
	print("nuuuts")
	Citizen.Wait(6000)
	for door_index,door in ipairs(doors) do
		Config.Doors[door_index] = door
		tempdist = Config.Doors[door_index].dist + 1
		namelort = Config.Doors[door_index].text_loc.x
		table.insert(doorsliste, namelort)
		print(Config.Doors[door_index].auth_jobs)
		if Config.Doors[door_index].auth_jobs[PlayerData.job.name] then
			exports.interact:RemoveInteraction(Config.Doors[door_index].text_loc.x)
			exports.interact:AddInteraction({
				coords = vec3(Config.Doors[door_index].text_loc.x, Config.Doors[door_index].text_loc.y, Config.Doors[door_index].text_loc.z),
				distance = tempdist, -- optional
				interactDst = Config.Doors[door_index].dist, -- optional
				id = namelort,
				options = {
					{
						label = 'Åben/Lås dør',
						action = function(entity, coords, args)
							print(entity, coords, json.encode(args))
						end,
					},
				}
			})
		end
	end
end

**Comment by Duckmc34** [comment]
I FIXED IT BECAUSE THESE ABSOLUTE BAFOONS WON'T, JUST GO TO CLIENT/INTERACTIONS.LUA AND FIND LINE 636 AND REPLACE YOUR CURRENT api.getNearbyInteractions() WITH THIS ONE:


function api.getNearbyInteractions()
    local options = {}
    local amount = 0

    local playercoords = GetEntityCoords(cache.ped)

    local amountOfInteractions = #filteredInteractions

    if amountOfInteractions > 0 then
        for i = 1, amountOfInteractions do
            local interaction = filteredInteractions[i]

            if interaction.global then
                addGlobalVehicleData(interaction, options, playercoords)
                amount = #options
                goto skip
            end

            if interaction.model then
                addModelInteraction(interaction, options, playercoords)
                amount = #options
                goto skip
            end

            -- Check if the interaction is a networked entity
            if interaction.netId then
                local entity = entities.isNetIdNearby(interaction.netId)

                if not entity then
                    goto skip
                end

                interaction.entity = entity
            elseif interaction.entity and not entities.isEntityNearby(interaction.entity) then
                goto skip
            end

            local coords = interaction.coords or utils.getCoordsFromInteract(interaction)
            local distance = #(coords - playercoords)

            if distance <= interaction.distance then
                local interactOptions, interactionAmount = getInteractionOptions(interaction)
                if interactionAmount > 0 then
                    amount += 1
                    options[amount] = getReturnData(interactOptions, distance, interaction)
                end
            end

            :: skip ::
        end
    end

    if amount > 1 then
        table_sort(options, function(a, b)
            return a.curDist < b.curDist
        end)
    end

    return options, amount
end

**Comment by FjamZoo** [comment]
its been reverted

**Comment by Duckmc34** [comment]
> Yo you think you so clever! Just revert the commit.. dont be a entitled bitch and cry about a free script in issues! @Duckmc34

You're embarrassing yourself, you are probably the one who came up with the silly feature.... The reason the script is free is because it isn't worth buying, make something worth selling before going bazinga mode, fool.

Thank you FjamZoo, have a great day.

**Comment by pushkart2** [comment]
Nobody fucking asked you to use the script! Making my point that you are a entitled bitch! Also come to me with your original account rather than opening a new account!

**Comment by Duckmc34** [comment]
"also come to me with your original account" holy hell, we can hear you drooling all over your keyboard pal, i don't use github.. i made this account directly to tell on your horrendous feature.

**Comment by CrackCommerce** [comment]
**@pushkart2 Wtf are you talking about. I came to this thread regarding the same issue. That feature is horrible!! Stick to .sk 
![maxresdefault](https://github.com/darktrovx/interact/assets/45232122/96a57122-7043-48df-9400-d3caea35cd68)
scripts you clown. How about you get your head out of your anus and stop blappering on github, when you are wrong!!!!**
